scss/_reboot.scss

```diff
@@ -443,13 +443,6 @@ select {
   }
 }
 
-// Remove the dropdown arrow in Chrome from inputs built with datalists.
-// See https://stackoverflow.com/a/54997118
-
-[list]::-webkit-calendar-picker-indicator {
-  display: none;
```

Comment:
We may improve this by setting `opacity: 0` instead of `display: none`. This way we'd still remove the Chrome arrow with working `datalist`.

FWIW the weird arrow is a `list-style: disclosure-open inside` UA style, could be replaced I guess.

Reply:
Yes, ```opacity: 0``` works. 

But on input types ```date```, ```time```, ```month```, etc, the datalist won't show up unless clicked on the indicator. So, if we hide it, the user won't know if he/she has to click on the extreme right of the field.

I have created a CodePen demo: https://codepen.io/nkdas91/pen/MWEXzvL where we can use ```opacity: 0``` for text type inputs and can replace the indicator with Bootstrap Icons if we want to.

Please advise. I'll make changes accordingly. Thanks!

Reply:
That's a really neat idea. Thanks!

I'd even go with a dropdown-like icon for text inputs, what do you think?

Another consideration is whether we should make it a default or not, since it'd be kinda heavy. We might revisit this when bundlewatch run.